### PR TITLE
Automate some of CLI gpg key stubs

### LINKED ISSUE #2480: Automate : gpg update in cli
please automate:
$ hammer gpg update --help
Usage:
    hammer gpg update [OPTIONS]

Options:
 --id ID                                 gpg key numeric identifier
 --key GPG_KEY_FILE                      GPG Key file
 --name NAME                             Name to search by
 --new-name NEW_NAME                     identifier of the gpg key
 --organization ORGANIZATION_NAME        Organization name to search by
 --organization-id ORGANIZATION_ID       organization ID
 --organization-label ORGANIZATION_LABEL Organization label to search by
 -h, --help                              print help

### LINKED ISSUE #2481: Automate : gpg delete in cli
please automate:
$ hammer gpg delete --help
Usage:
    hammer gpg delete [OPTIONS]

Options:
 --id ID                                 gpg key numeric identifier
 --name NAME                             Name to search by
 --organization ORGANIZATION_NAME        Organization name to search by
 --organization-id ORGANIZATION_ID       organization ID
 --organization-label ORGANIZATION_LABEL Organization label to search by
 -h, --help                              print help

### PR DESCRIPTION
**Pull request by abalakh**
Automated some of CLI gpg key stub and updated existing tests.
Closes #2480
Closes #2481 
```python
py.test tests/foreman/cli/test_gpgkey.py
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.2, py-1.4.30, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
collected 64 items 

tests/foreman/cli/test_gpgkey.py sss...s..s...s...sssssssss.ssssssssssssss..s..ss..ss.sss.s..ss.s

====================== 23 passed, 41 skipped in 1033.30 seconds ======================
```

### CODE REVIEW

**Comment by oshtaier** [comment]
If you don't mind, could you please fix test names according to https://github.com/SatelliteQE/robottelo/issues/3083 (I know that it is not still approved, but it will not be seriously different from current form)

**Comment by oshtaier** [comment]
Please, mark all tests with corresponding `tierN` decorators

**Comment by abalakh** [comment]
@oshtaier i left naming old style on purpose. There're 64 tests, and all of them are named like `test_key_associate_34`. It will take some extra effort to rename that mess. I can create a separate issue to track that if you wish.

**Comment by abalakh** [comment]
@oshtaier niiice :+1:  sure, will update with tiers.

**Comment by oshtaier** [comment]
@abalakh please, rename basic ones and create separate issue to track `associate_N`.

**Comment by abalakh** [comment]
@oshtaier makes sence, will do that.

**Comment by elyezer** [comment]
@oshtaier good suggestions :+1:

**Comment by abalakh** [comment]
Added tiers & updated test names. Submitted #3087 to track `test_key_associate_NN` names.

File: tests/foreman/cli/test_gpgkey.py [hunk 1]
```diff
@@ -308,103 +293,83 @@ def test_negative_delete_1(self):
...
         @assert: gpg key is updated
-
-        @status: manual
-        """
-
-    @stubbed()
-    @run_only_on('sat')
-    def test_positive_update_3(self):
-        """@test: Create gpg key with valid name and valid gpg key text via
-        cut and paste/string then update its name
```
Comment (abalakh) [inline tests/foreman/cli/test_gpgkey.py]: Deleted such stubs as we can't copypaste key via CLI. CLI only supports attaching key file:
```
Usage:
    hammer gpg create [OPTIONS]

Options:
 --key GPG_KEY_FILE                      GPG Key file
 --name NAME                             identifier of the gpg key
 --organization ORGANIZATION_NAME        Organization name to search by
 --organization-id ORGANIZATION_ID       organization ID
 --organization-label ORGANIZATION_LABEL Organization label to search by
```

**Comment by oshtaier** [comment]
We don't have final decision whether use pos or positive (neg or negative), but definitely not a blocker

ACK

**Comment by elyezer** [comment]
ACK

**Pull request merged**